Gemfile

```diff
@@ -21,7 +21,6 @@ gem "jbuilder", "~> 2.7"
 # Use Redis adapter to run Action Cable in production
 # gem 'redis', '~> 4.0'
 # Use Active Model has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
```

Comment:
why you comment it ?

Reply:
I follow the video tutorial. Should I remove it?

Reply:
nope, you need to un comment this line
```
gem 'bcrypt', '~> 3.1.7'
```

Reply:
please  uncomment this line
`gem 'bcrypt', '~> 3.1.7'`